config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      post 'auth/login', to: 'authentication#login'
+      post 'auth/register', to: 'authentication#register'
+      delete 'auth/logout', to: 'authentication#logout'
+      
+      resources :users
+      resources :reservations
+      resources :cars
```

Comment:
- Please, add `only:` statement here, this works to avoid the not used routes open in the project. 🐱 

Example (something like this):

`resources :cars :only[:index :show...]`